src/utils/hooks/useLoadApplication.tsx

```diff
@@ -24,20 +28,31 @@ const useLoadApplication = (props: UseGetApplicationProps) => {
   })
 
   useEffect(() => {
-    if (data && data.applicationBySerial && data.applicationStageStatusAlls) {
+    if (data?.applicationBySerial) {
       const application = data.applicationBySerial as Application
 
-      const applicationDetails = {
+      let applicationDetails: ApplicationDetails = {
```

Comment:
Why change to `let` here?

Reply:
To update the stage (if found) after:
https://github.com/openmsupply/application-manager-web-app/blob/48c8899fe539e62ad8392c552f06784b0aa9a16c/src/utils/hooks/useLoadApplication.tsx#L48-L53

Reply:
But it's fine to mutate a property of a `const`, it only needs `let` if you're reassigning the reference to the object itself.

Reply:
Oops! That's a good point 😆 I'll change to const : )